cip/1.accepted/CIP2015-09-16-public-type-system-type-annotation.adoc

```diff
@@ -54,26 +54,35 @@ The section below presents presents the new Public Type System (PTS) proposed fo
 
 ==== Material and nullable types
 
-The type system provides a way to track nullability, i.e. a type may express if a given expression may be `NULL` or not.
-Tracking (non-)nullability helps an implementation to detect and warn about possible user errors during compile time and  it also enables optimizations that rely on values not being `NULL`. As an example, an implementation may infer that `MATCH` will never introduce a nullable variable, but `OPTIONAL MATCH` will only introduce nullable variables.
+The type system provides a way to track nullability, i.e. a type may express if a given expression may be `null` or not.
+Tracking (non-)nullability helps an implementation to detect and warn about possible user errors during compile time and  it also enables optimizations that rely on values not being `null`. As an example, an implementation may infer that `MATCH` will never introduce a nullable variable, but `OPTIONAL MATCH` will only introduce nullable variables.
 
-Material types are all types that do not permit `NULL` as a valid result of evaluating the underlying expression.
+Material types are all types that do not permit `null` as a valid result of evaluating the underlying expression.
 In the sections below, generally only the material variants of each type are given.
 
-Nullable types are all types that do permit `NULL` as a valid result of evaluating the underlying expression.
+Nullable types are all types that do permit `null` as a valid result of evaluating the underlying expression.
 Nullable type names are formed by suffixing a material type name with a single question mark (without in-between whitespace).
 Nullable variants of all material types should be supported by any implementation of Cypher.
-Note that `NULL` is not a type but a value that inhabits every nullable type.
+Note that `null` is not a type but a value that inhabits every nullable type.
 
 Every nullable type is a supertype of its underlying material type, i.e. `T?` is a supertype of `T`.
 By transitivity of subtyping, if `S` is a supertype of `T`, then `S?` is also a supertype of `T`.
 
+==== Abstract and concrete types
+
+An _abstract_ type is a type that does not have any direct association to a value.
+This is in contrast to a _concrete_ type, which does have such an association.
+In other words, an abstract type may not be instantiated, while a concrete type may.
+
+It is observed that any nullable type is also concrete, because `null` is a value that is directly associated with it.
```

Comment:
"It is observed that any nullable type is also concrete, because `null` is a value that is directly associated"
-->
"We observe that any nullable type is also concrete as `null` is a value that is directly associated"